unabridged-works-volume-4/original/02-life-and-immortality.adoc

```diff
@@ -4725,7 +5037,7 @@ which cannot really satisfy that which is indeed begotten and born of God`'s Spi
 
 ==== XXIX. A brief Account of the Ground of our Worship, and how it cometh to pass that we cannot conform to the spirit of this world, or to the wills of men therein, but only to the Spirit and Will of our God
```

Comment:
this should have been labeled `[.inline]` and so should the `XXX.` one on line 5125, can you add those.  that was my bad, sorry!